psalm.xml

```diff
@@ -1,17 +1,16 @@
 <?xml version="1.0"?>
 <psalm
-    errorLevel="8"
+    errorLevel="7"
```

Comment:
wondered why we use the lowest possible error level in psalm. So bump it up to 7 and fixed couple new violations